### ISSUE TITLE
Accessibility Issue : Links do not have a discernible name

### ISSUE THREAD

**Issue by onwordi** [Oliversw/oliversw.github.io#1]
Missing link text (and alternate text for images (**_alt_**), when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users.

```<a target="_blank" href="https://www.facebook.com/oliversw">
<a target="_blank" href="https://twitter.com/Oliversw">
<a target="_blank" href="https://github.com/Oliversw">
<a target="_blank" href="https://www.linkedin.com/in/oliver-smith-wellnitz-15817982/">
